charts/actions-runner-controller/templates/githubwebhook.ingress.yaml

```
@@ -36,6 +36,9 @@ spec:
     - host: {{ .host | quote }}
       http:
         paths:
+          {{- if .extraPaths }}
+          {{ toYaml .extraPaths | indent 10 }}
```

@KENNYSOFT Hey! Just wondering, but doesn't this result in the first extraPaths item to be rendered with 10 + 10 = 20 spaces indentation? Perhaps you meant `nindent 10`, right?
```suggestion
          {{ toYaml .extraPaths | nindent 10 }}
```

---

Sorry for the late, just confirmed by using command:

```bash
helm template --debug --set githubWebhookServer.ingress.enabled=true --set githubWebhookServer.ingress.hosts[0].extraPaths.path="/*" .
```

result:

```yaml
# Source: actions-runner-controller/templates/githubwebhook.ingress.yaml
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: RELEASE-NAME-actions-runner-controller-github-webhook-server
  labels:
    helm.sh/chart: actions-runner-controller-0.16.0
    app.kubernetes.io/name: actions-runner-controller
    app.kubernetes.io/instance: RELEASE-NAME
    app.kubernetes.io/version: "0.21.0"
    app.kubernetes.io/managed-by: Helm
spec:
  rules:
    - host:
      http:
        paths:
                    path: /*
```

😱